components/server/src/workspace/workspace-starter.ts

```diff
@@ -1251,6 +1251,11 @@ export class WorkspaceStarter {
             ideImage = ideConfig.ideOptions.options[ideConfig.ideOptions.defaultIde].image;
         }
 
+        let workspaceClass: string = "";
```

Comment:
Can we set "default" instead of an empty string? Same issue with labels, we need a value :(

Reply:
`let workspaceClass: string = "default";` ?

Reply:
Would suggest `standard` as we have been using that name already

Reply:
> Would suggest standard as we have been using that name already

We already use that in [code (go)](https://github.com/gitpod-io/gitpod/blob/3939796825c604fe1e3fa6c9a59993a8abbb1ce6/components/ws-manager-api/go/config/config.go#L29) and also the [ops](https://github.com/gitpod-io/ops/blob/45b7d2ea558a4716e1f8fdbd149391f98096f4e9/deploy/workspace/manifests/workspace-templates.yaml#L12) repo